rllib/policy/sample_batch.py

```diff
@@ -1619,8 +1623,20 @@ def concat_samples_into_ma_batch(samples: List[SampleBatchType]) -> "MultiAgentB
     return MultiAgentBatch(out, env_steps)
 
 
-def _concat_key(*values, time_major=None):
-    return concat_aligned(list(values), time_major)
```

Comment:
can you share why we don't need to do this array alignment anymore?

Reply:
@gjoliver Based off a discussion with @avnishn :
`concat_aligned` has been added to this repo several years ago. This alignment strategy was introduced to make tf quicker in moving data to a cache. But since the need to do this is not obvious to us anymore (probably because tf has progressed), we expect such optimizations to be made efficiently on the side of TensorFlow if need be.
The idea here is to remove code for which we don't understand why it's there if it does not result in a visible regression.